spec/services/expire_requestable_spec.rb

```diff
@@ -18,6 +18,11 @@
           creator: user,
         )
       end
+
+      it "calls the status updater" do
+        expect(ApplicationFormStatusUpdater).to receive(:call).at_least(:once)
```

Comment:
```suggestion
        expect(ApplicationFormStatusUpdater).to receive(:call).with(application_form: requestable.application_form, user:).at_least(:once)
```

To just check we're passing the right arguments in.

Reply:
Should be in now